tests/jenkins/jenkinsjob-regression-files/docker-ecr/docker-build-with-dockerhub-only.jenkinsfile.txt

```diff
@@ -0,0 +1,22 @@
+   docker-build-with-ecr.run()
+      docker-build-with-ecr.legacySCM(groovy.lang.Closure)
+      docker-build-with-ecr.library({identifier=jenkins@20211123, retriever=null})
+      docker-build-with-ecr.pipeline(groovy.lang.Closure)
+         docker-build-with-ecr.timeout({time=3, unit=HOURS})
```

Comment:
Needs to be updated to 1 hour

Reply:
Good spot. I missed committing these files.